### PR TITLE
support deployment of thoth app graph-schema-update with argo

### PR DESCRIPTION
**Pull request by harshad16**
## This Pull Request implements

support deployment of thoth app graph-schema-update with argo
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Related-to: https://github.com/thoth-station/thoth-application/pull/687

## If migrating an Application to ArgoCD
- [x] I have completed the list of action items on [this page](https://aicoe.github.io/aicoe-cd/get_argocd_to_manage_your_app/)

### CODE REVIEW

File: manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-graph-schema-update.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    repoURL: 'https://github.com/thoth-station/thoth-application.git'
+    path: graph-schema-update/overlays/ocp4-stage
+    targetRevision: master
+  destination:
+    server: 'https://api.ocp4.prod.psi.redhat.com:6443'
+    namespace: thoth-infra-stage
+  syncPolicy:
+    automated:
+      selfHeal: true
+      prune: true
```
Comment (anishasthana) [inline manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-graph-schema-update.yaml:18]: You may need to disable validation as well. ArgoCD usually fails on some nitpicks with OpenShift objects

Comment (harshad16) [reply manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-graph-schema-update.yaml:18]: @anishasthana was the comment related to the `validate` option under `syncOptions`? I was under the impression that it is disabled my default.

Comment (anishasthana) [reply manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-graph-schema-update.yaml:18]: I was of the impression that it is enabled by default. @HumairAK thoughts?

Comment (HumairAK) [reply manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-graph-schema-update.yaml:18]: Hrmm honestly I'm not sure, can't find anything in the docs. I would just add it to eliminate doubt.

Comment (harshad16) [reply manifests/overlays/prod/applications/thoth-station/ocp4-stage/stage-thoth-graph-schema-update.yaml:18]: ok , thanks both of you. update it as well.

**Review by anishasthana — APPROVED** [review]

**Pull request merged**